### ISSUE TITLE
Public route for unsubscribe (sent in emails)

### ISSUE THREAD

**Issue by diego3g** [diego3g/umbriel#85]
Allow contact to unsubscribe by clicking on the `unsubscribe` link inside the email.

**Comment by diego3g** [comment]
We'll need a page so the user can confirm the unsubscription.

**Comment by brunooomelo** [comment]
Do you think about using some template engine for page?

**Comment by diego3g** [comment]
Yeah, I think we could use handlebars to keep it simple, but it's not a big deal if we choose some other library.
